README.md

```diff
@@ -88,7 +88,7 @@ cmake --build build --target all -j 4
 
 #### Dependencies
 ```cmd
-vcpkg install --triplet x64-windows zlib boost-system boost-program-options boost-test boost-align boost-foreach boost-math boost-uuid cpprestsdk flatbuffers
+vcpkg install --triplet x64-windows zlib boost-system boost-program-options boost-locale boost-test boost-align boost-foreach boost-math boost-uuid cpprestsdk flatbuffers
```

Comment:
Why is pulling in a new dependency required?

Reply:
To support Token auth new changes are implemented to allow HTTP header name from config. In order to convert it to http_headers::key_type, wstring conversion is required. Boost locale is supported in both linux and windows, but as it's not currently available in our project have to add it in dependency. Tried using codecvt but checks are failing in linux. Looking for other alternatives

Reply:
What is the type of `http_headers::key_type`? It seems to be std::string based on the docs: [1](https://microsoft.github.io/cpprestsdk/classweb_1_1http_1_1http__headers.html#a00428d8e7a1ee544227efdae54550835) [2](https://microsoft.github.io/cpprestsdk/namespaceutility.html)

Is the conversion from or to wide string?